File: week02/2_Robot1000/2_Robot1000.py
walk = input()
i = 0
x = 0
y = 0
while (i < len(walk)):
    if walk[i] == 'Z':
        x = 0
        y = 0
    elif walk[i] == 'E':
        x = x + 1
    elif walk[i] == 'W':
        x = x - 1
    if walk[i] == 'N':
        y = y + 1
    elif walk[i] == 'S':
        y = y - 1
    i += 1
print(x,y)

File: week02/2_Robot1000/2_Robot1000.cpp
#include <iostream>
using namespace std;

int main()
{
    string direction;
    int x=0,y=0;
    cin >> direction;
    for(int i=0;i<direction.size();i++){
        if(direction[i] == 'N'){
            y++;
        }
        else if(direction[i] == 'S'){
            y--;
        }
        else if(direction[i] == 'E'){
            x++;
        }
        else if(direction[i] == 'W'){
            x--;
        }
        else if(direction[i] == 'Z'){
            x = 0;
            y = 0;
        }
    }
    cout << x << " " << y << endl;
    // input
    // NNEESW : 1 1
    // NNEESWZEE : 2 0
    // NWSSSSE : 0 -3
}